pkg/framework/env.go

```diff
@@ -4,32 +4,52 @@ import (
 	"github.com/spf13/viper"
 )
 
-type Env struct {
-	LogLevel    string `mapstructure:"LOG_LEVEL"`
+type AdminConfig struct {
+	Email    string `mapstructure:"ADMIN_EMAIL"`
+	Password string `mapstructure:"ADMIN_PASSWORD"`
+}
+
+type AWSConfig struct {
+	Region            string `mapstructure:"AWS_REGION"`
+	AccessKey         string `mapstructure:"AWS_ACCESS_KEY_ID"`
+	SecretAccessKey   string `mapstructure:"AWS_SECRET_ACCESS_KEY"`
+	StorageBucketName string `mapstructure:"STORAGE_BUCKET_NAME"`
+	ClientID          string `mapstructure:"COGNITO_CLIENT_ID"`
+	UserPoolID        string `mapstructure:"COGNITO_USER_POOL_ID"`
```

Comment:
Let's include Cognito in this attributes
For eg: CognitoClientID, CognitoUserPoolD